examples/Pin_control/Pin_control.ino

```diff
@@ -5,57 +5,176 @@
+      If there is a USB power pin exposed, that should be safe.
+      Note, on the Uno and Mega 2560 that pin is call "5V".
+      Otherwise, just supply a separate USB power cable to the Target.
+
+   How To:
+    This sketch is written with Uno pins in mind. 
+    You may have to change the pin numbers for other Arduinos.
+    
+    Pick 2 Arduinos. One will be the Controller, and one the Target.
+    Pick Arduinos that have the same `Vcc`, ie. 3v or 5v `logic level`.
+      (or you'll have to use a logic-level shifter) 
+    And, the simplest switch: single-throw (or momentary), normally-open.
+
+    Setup
+    1. If you know the right pins on the Target, and you aren't using the default pins, 
+      edit this sketch and adjust the pin numbers for your Arduino.
+      Otherwise, you could do "Prove Target" (below) and use the first bit of output to help you.
+
+    Prove Target
+    1. Wire up the Target as above, but do not connect the I2C wires
+    2. Do not connect Vcc/Vusb to the Controller
+    3. Uncomment `#define I2CWRAPPER_LOOPBACK`
+    4. Upload this sketch to the Target
+       Watch for any "static assertion failed" erorrs, 
+       this sketch can detect some cases where aPinIn and aPinOut are not valid.
+    5. Open the console
+       Note the first bit, it tries to give you a clue about analog-pins and LED_BUILTIN.
+    Be patient with the following tests, the inputs are only tested every 1/2 second, so slow response
+    6. Test the switch
+       The `dPinOut` LED (builtin on Uno) should follow the switch open/closed.
+       And the console will echo it.
+    7. Test the potentiometer
+       The `aPinOut` LED (e.g. PWM pin 6) should follow the potentiometer in brightness.
+       And the console will echo it.
+    8. Debug wiring if the LEDs or console in the above doesn't "follow" the hardware.
+       And/or figure out the correct pin INTEGERS and adjust this sketch.
+
+    Setup I2C: Controller & Target as target
+    1. Open examples:i2cwrapper:firmware
+    2. Edit firmware_modules.h, uncomment the line: #include "PinI2C_firmware.h"
+    3  Save
+    4. Upload to the Target
+    5. check console? set debug?
+    6. Disconnect the Target from USB
+    7. Wire the Controller to Target with I2C
+       Don't forget GND to GND
+    8. Wire Vcc together.
+       CAUTION: only if voltage is the same (3v vs 5v!)
+       CAUTION: only if no USB is connected to the Target
+       CAUTION: only if you know what Vcc's to connect.
+       (or, no Vcc connection and using USB power on Target, still need GND to GND)
+    9. Comment `#define I2CWRAPPER_LOOPBACK`
+    10. Upload this sketch to the Controller
+    11. Open the console
+    12. Test the hardware on the Target just like "Prove Target" above!
+        Same output to console!
+    13. There should be helpful messages at each stage,
+        and that should help you figure out if something hangs or goes wrong.
 */
 
+//#define I2CWRAPPER_LOOPBACK
```

Comment:
If defined, then use `PinI2C_LoopBack` as the class, which just maps methods to the *local* arduino methods. I.e. talk to myself, no I2C.

This is useful to see if the code/wiring works. If you already had written a controler that relied on outsourcing PinI2C, this might not be a useful strategy. But, I often write skeleton programs to test 1 piece of hardware at a time, and this would allow:

* Test code/wiring without I2C (especially pin numbers)
* Test with I2C, with the same code